### ISSUE TITLE
Copy Paste in Tabs Navigator

### ISSUE THREAD

**Issue by ibnekhan** [petrogad/react-native-materialui-textfield#2]
Have you fixed that copy paste issue in tabs navigator?

**Comment by petrogad** [comment]
@ibnekhan - Can you be a bit more descriptive about what you're referring to here? Not sure I follow.

**Comment by ibnekhan** [comment]
> @ibnekhan - Can you be a bit more descriptive about what you're referring to here? Not sure I follow.

I have used this component in react native tabs and facing an issue in copy/paste selection.
Not able to select, copy or paste text in materialui textfield when i used inside tabs. 
And this happens only on ANDROID, on iOS its working.

**Comment by petrogad** [comment]
@ibnekhan have you tried this on a physical device not the simulator? I've noticed the copy/paste on an android simulator is rather fragile to the point where it's exhibited different behaviors from my computer to co-worker computers.

I'm assuming you're referring to react-navigation's tabNavigator and inside one of the tab stacks, a screen using a select field can't be copied/pasted. Is this correct?

Can you confirm that it's not working on an android device?

Thanks

**Comment by ibnekhan** [comment]
@petrogad Yes i have tested on physical devices and not able to copy/paste. 
So far i have tested within react-navigation tabNavigator and facing that issue on android devices.

**Comment by petrogad** [comment]
Curious, I'll see if I can spin up an example and look to see what can be done to solve it.

@ibnekhan
